### PR TITLE
Added a pending cases endpoint, data science judge endpoint and moved…

### PR DESCRIPTION
**Pull request by Poonchy**
… some endpoint to fix some bugs

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change
- Added endpoint for datascience
- Added endpoint for pending cases

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ The base technologies are JavaScript, HTML and CSS. The frontend leverages [Reac
 | GET    | `/cases/:id/download-pdf` | -                | `PDF file of the case`           |
 | GET    | `/cases/:id/download-csv` | -                | `case information as CSV format` |
 | PUT    | `/cases/:id`              | `Reference case` | `{Reference case}`               |
+| GET    | `/cases/pending/user/:id` | -                | `[{Reference cases}]`            |
```
Comment (nicholasbus) [inline README.md:71]: Nice documentation 👍

File: api/cases/caseModel.js [hunk 2]
```diff
@@ -36,8 +36,17 @@ const findBy = async (filter) => {
...
+    .where({ user_id })
+    .where({ status: 'approved' })
+    .join('judges as j', 'j.judge_id', 'c.judge_id')
+    .select('c.*', 'j.first_name', 'j.middle_initial', 'j.last_name');
+};
+
+const findPendingByUserId = (user_id) => {
+  return db('cases as c')
     .where({ user_id })
+    .whereNot({ status: 'approved' })
```
Comment (nicholasbus) [inline api/cases/caseModel.js:49]: I like the "Not approved" differentiation with the status flag.

**Review by nicholasbus — APPROVED** [review]
All endpoints look good, I like the documentation and the optimization of the pending cases.

**Review by lindellcarternyc — APPROVED** [review]
Great job. I really appreciate the naming conventions and attention to detail!

**Pull request merged**